iree/samples/simple_embedding/CMakeLists.txt

```diff
@@ -243,6 +243,44 @@ iree_run_binary_test(
     ::simple_embedding_vulkan
 )
 
+iree_cc_binary(
```

Comment:
this needs to be wrapped in its own if(...DRIVER_CUDA), not the vulkan one

Reply:
fixed